sites/partners/src/listings/PaperListingDetails/sections/DetailApplicationAddress.tsx

```diff
@@ -14,6 +15,10 @@ const DetailApplicationAddress = () => {
       return t("application.contact.mailingAddress")
   }
 
+  const postMarkDateFormat = (date: Date) => {
+    return date ? moment(new Date(date)).format("MM/DD/YYYY") : t("t.none")
```

Comment:
We have code very similar to this in other places, but I think we can handle that when we do the moment -> dayjs switch (#1290)